spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JpqlQueryRenderer.java

```diff
@@ -142,12 +143,8 @@ public List<JpaQueryParsingToken> visitIdentification_variable_declaration(
 
 		tokens.addAll(visit(ctx.range_variable_declaration()));
 
-		ctx.join().forEach(joinContext -> {
-			tokens.addAll(visit(joinContext));
-		});
-		ctx.fetch_join().forEach(fetchJoinContext -> {
-			tokens.addAll(visit(fetchJoinContext));
-		});
+		ctx.join().forEach(joinContext -> tokens.addAll(visit(joinContext)));
+		ctx.fetch_join().forEach(fetchJoinContext -> tokens.addAll(visit(fetchJoinContext)));
 
```

Comment:
I am usually a fan of shortening things like this, but since this is an ANTLR Visitor implementation, I actually prefer to retain it the way it is. It makes it easy when grammar tweaks require me to add another `tokens.addAll(...)` inside that clause. And since this pattern is used A LOT in this class (and its counterpart), it becomes easier to maintain if I retain the pattern consistently everywhere.